src/main/java/de/hpi/bpt/argos/storage/hierarchy/HierarchyBuilder.java

```diff
@@ -0,0 +1,13 @@
+package de.hpi.bpt.argos.storage.hierarchy;
+
+/**
+ * This interface offers methods to build the entityHierarchy.
+ */
+public interface HierarchyBuilder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Annotate the "HierarchyBuilder" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1609)